internal/cli/command/cluster/import.go

```diff
@@ -0,0 +1,183 @@
+	orgId := banzaiCli.Context().OrganizationID()
+
+	if banzaiCli.Interactive() {
+		var err error
+		options.kubeconfig, options.name, err = buildInteractiveImportRequest(banzaiCli, options, orgId)
+		if err != nil {
+			return err
+		}
+		options.kubeconfig = base64.StdEncoding.EncodeToString([]byte(options.kubeconfig))
+	} else {
+		filename, raw, err := utils.ReadFileOrStdin(options.file)
+		if err != nil {
+			return emperror.WrapWith(err, "failed to read", "filename", filename)
+		}
+		options.kubeconfig = base64.StdEncoding.EncodeToString(raw)
+	}
+
+	// Create kubernetes secret
+	req := pkgPipeline.CreateSecretRequest{}
+	req.Name = options.name
+	req.Type = "kubernetes"
+	req.Values = make(map[string]interface{}, 1)
+	req.Values["K8Sconfig"] = options.kubeconfig
+
+	// Validate secret create request
+	if err := validateSecretCreateRequest(req); err != nil {
+		return emperror.Wrap(err, "failed to create cluster request")
+	}
+
+	req.Name += "-kubeconfig"
+
+	if bytes, err := json.MarshalIndent(req, "", "  "); err != nil {
+		log.Errorf("failed to marshal request: %v", err)
+		log.Debugf("Request: %#v", req)
+	} else {
+		_, _ = fmt.Fprintf(os.Stderr, "The current state of the request:\n\n%s\n", bytes)
+	}
+
+	secret, resp, err := pipeline.SecretsApi.AddSecrets(context.Background(), orgId, req, nil)
+	if err != nil {
+		// Secret already exists
+		if resp != nil && resp.StatusCode == http.StatusConflict {
+			return emperror.WrapWith(err, "secret with this name is already created", "secret", req.Name)
+		}
+		// Generic error
+		if oerr, ok := err.(pkgPipeline.GenericOpenAPIError); ok {
+			log.Errorf("secret creation error: %s", oerr.Body())
+		}
+
+		return emperror.Wrap(err, "failed to create Kubernetes secret")
+	}
+
+	log.Debugf("Kubernetes config secret created with id: %s\n", secret.Id)
+
+	// Create cluster with kubernetes secret
+	body := make(map[string]interface{}, 4)
+	body["name"] = options.name
+	body["secretId"] = secret.Id
+	body["cloud"] = "kubernetes"
+	prop := make(map[string]interface{}, 1)
+	prop["kubernetes"] = make(map[string]interface{}, 0)
+	body["properties"] = prop
+
+	if bytes, err := json.MarshalIndent(body, "", "  "); err != nil {
+		log.Errorf("failed to marshal request: %v", err)
+		log.Debugf("Request: %#v", req)
+	} else {
+		_, _ = fmt.Fprintf(os.Stderr, "The current state of the request:\n\n%s\n", bytes)
+	}
+
+	cluster, _, err := pipeline.ClustersApi.CreateCluster(context.Background(), orgId, body)
+	if err != nil {
+		// Generic error
+		if oerr, ok := err.(pkgPipeline.GenericOpenAPIError); ok {
+			log.Errorf("secret creation error: %s", oerr.Body())
+		}
+
+		return emperror.Wrap(err, "failed to import Kubernetes cluster")
+	}
+
+	log.Debugf("Kubernetes config secret created with id: %d\n", cluster.Id)
+
+	return nil
+}
+
+func buildInteractiveImportRequest(banzaiCli cli.Cli, options importOptions, orgId int32) (kubeconfig, clusterName string, err error) {
+	_ = banzaiCli
```

Comment:
What's the point of this? If those arguments are not used and the linter complains, you can just replace the variable name in the function definition with the ignore operator.